standup_notes/__main__.py

```diff
@@ -1,33 +1,35 @@
+# PYTHON_ARGCOMPLETE_OK
 import os
 import sys
-from argparse import ArgumentParser, Namespace
 from datetime import date, timedelta
 from typing import Callable
 
-from pkg_resources import resource_stream
-
+import argcomplete
+import argparse
 import editor
 import pyperclip
+from pkg_resources import resource_stream
 
 EXT = '.standup-notes.txt'
 STANDUP_NOTES = os.path.join(os.environ.get("HOME"), 'Desktop/standup-notes')
 STANDUP_TEMPLATE = resource_stream('standup_notes.resources', 'standup.template')
 
 
 def main():
-    parser = ArgumentParser()
-    parser.add_argument('--list', help='List all stand-up notes.', action='store_true')
-    parser.add_argument('--read-yesterday', help='Read yesterday\'s stand-up notes.', action='store_true')
-    parser.add_argument('--read-today', help='Read today\'s stand-up notes.', action='store_true')
-    parser.add_argument('--read-tomorrow', help='Read tomorrow\'s stand-up notes.', action='store_true')
-    parser.add_argument('--copy-yesterday', help='Copy yesterday\'s stand-up notes to the clipboard.',
-                        action='store_true')
-    parser.add_argument('--copy-today', help='Copy today\'s stand-up notes to the clipboard.', action='store_true')
-    parser.add_argument('--copy-tomorrow', help='Copy tomorrow\'s stand-up notes to the clipboard.',
+    parser = argparse.ArgumentParser()
+    days = parser.add_mutually_exclusive_group()
```

Comment:
what happens if no day is passed in?

Reply:
Nothing

Reply:
Could you add a check to make sure a day is passed in if copy, edit, or read is passed in (i.e. throw an error if one wasn't specified)? Also what happens if a day is passed in with something like "delete", where it's not expecting a day?

Reply:
Print statement remind user to make sure arguments are correct and refers them to the help section. If a day is passed into delete, it errors out and tells the user it expected an argument